src/sensors/bmi160sensor.cpp

```diff
@@ -271,17 +271,16 @@ void BMI160Sensor::startCalibration(int calibrationType) {
     delay(1500);
     m_Logger.debug("Gathering accelerometer data...");
 
+    MagnetoCalibration *magneto = new MagnetoCalibration();
+
     uint16_t accelCalibrationSamples = 300;
-    float *calibrationDataAcc = (float*)malloc(accelCalibrationSamples * 3 * sizeof(float));
     for (int i = 0; i < accelCalibrationSamples; i++)
     {
         ledManager.on();
 
         int16_t ax, ay, az;
         imu.getAcceleration(&ax, &ay, &az);
-        calibrationDataAcc[i * 3 + 0] = ax;
-        calibrationDataAcc[i * 3 + 1] = ay;
-        calibrationDataAcc[i * 3 + 2] = az;
+        magneto->sample(ax, ay, az);
```

Comment:
Can we also use this class to calibrate the acceleration?

Reply:
Yes. I went into more detail in my other comment. Moreover, in my opinion for this PR, it's less of a matter of whether we *can* and more that we *already did*.